apps/condo/domains/common/components/Import/Index.tsx

```diff
@@ -142,80 +155,217 @@ const ImportWrapper: React.FC<IImportWrapperProps> = (props) => {
+                        }
+                        return String(cell.value)
+                    })
+                    line.push(erroredRows[i].errors ? erroredRows[i].errors.join(', \n') : null)
+                    data.push(line)
+                }
+
+                const wb = XLSX.utils.book_new()
+                const ws = XLSX.utils.aoa_to_sheet(data)
+                ws['!cols'] = fitToColumn(data)
+                XLSX.utils.book_append_sheet(wb, ws, 'table')
+                XLSX.writeFile(wb, `${domainName}_failed_data.xlsx`)
+            } catch (e) {
+                reject(e)
+            } finally {
+                resolve()
+            }
+        })
+    }, [columns, domainName])
+
+    const closeModal = useCallback(() => setActiveModal(null), [])
+
+    const handleBreakImport = useCallback(() => {
+        closeModal()
+        breakImport()
+        if (isFunction(handleFinish)) {
+            handleFinish()
+        }
+    }, [breakImport, closeModal, handleFinish])
 
     return (
         accessCheck && (
-            <ModalContext.Provider value={{ progress, error, isImported }}>
-                <DataImporter onUpload={handleUpload}>
-                    <Popover
-                        title={<Typography.Text style={{ fontWeight: 'bold' }}>{ImportPopoverTitle}</Typography.Text>}
-                        content={<ColumnsInfoBox
-                            columns={columns}
-                            exampleTemplateLink={exampleTemplateLink}
-                            domainTranslate={domainTranslate}
-                        />}
-                    >
-                        {props.children}
-                    </Popover>
-                </DataImporter>
-                {contextHolder}
-            </ModalContext.Provider>
+            <>
+                <Button
+                    type='secondary'
+                    icon={<FileDown size='medium'/>}
+                    onClick={() => setActiveModal('example')}
+                >
+                    {UploadButtonLabel}
+                </Button>
+                <Modal
+                    title={UploadModalTitle}
+                    onCancel={closeModal}
+                    open={activeModal === 'example'}
+                    footer={
+                        <DataImporter onUpload={handleUpload}>
+                            <Button type='primary'>
+                                {ChooseFileForUploadLabel}
+                            </Button>
+                        </DataImporter>
+                    }
+                >
+                    <Space direction='vertical' size={24}>
+                        <Typography.Text>
+                            {UploadModalMessage}
+                        </Typography.Text>
+                        <ImageContaier>
+                            <img alt='example-import-image' src={exampleImageSrc} />
+                        </ImageContaier>
```

Comment:
Does it look good if the picture was not found in the specified path? For example, with `EN` locale

Reply:
Here will be a picture